code/modules/surgery/bodyparts/head.dm

```diff
@@ -119,6 +119,9 @@
 				for(var/datum/action/item_action/hands_free/activate_pill/AP in I.actions)
 					qdel(AP)
 			I.forceMove(T)
+	eyes = null
```

Comment:
this should all be handled in the above for statement

Reply:
As far as I can tell that would just do the same thing, but repeated as many times as there are items in the head.
Unless you mean creating a special case for each organ like there is for the brain, but since all the head's organs are getting dumped out anyways I don't see the point.